plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeRegisterTableProcedureUsingHiveMetastore.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.hive.metastore.HiveMetastore;
+
+import java.io.File;
+import java.util.Map;
+
+import static io.trino.plugin.hive.metastore.file.FileHiveMetastore.createTestingFileHiveMetastore;
+
+public class TestDeltaLakeRegisterTableProcedureUsingHiveMetastore
+        extends BaseDeltaLakeRegisterTableProcedureTest
+{
+    @Override
+    protected Map<String, String> getConnectorProperties(String dataDirectory)
+    {
+        return ImmutableMap.of(
+                "hive.metastore", "file",
```

Comment:
"UsingHiveMetastore" could be understood as "using HMS", ie. the thrift metastore.
In fact, it should be quite easy to do that.

Reply:
`TestDeltaLakeAdlsConnectorSmokeTest` and `TestDeltaLakeGcsConnectorSmokeTest` already cover test cases using thrift metastore.

So I will keep the current class with `file` metastore. I will rename the test class `TestDeltaLakeRegisterTableProcedureWithFile`. Make sense?

Reply:
I renamed to `TestDeltaLakeRegisterTableProcedureWithFileMetastore`